app/controllers/dogs_controller.rb

```diff
@@ -10,26 +10,10 @@ class DogsController < ApplicationController
   before_filter :admin_user,                              :only => [:destroy]
 
   def index
-    if (signed_in?) && (session[:mgr_view] == true)
-      @title = "Dog Manager"
-      if (params[:search].to_i > 0)
-        @dogs = Dog.where('tracking_id = ?', "#{params[:search].to_i}").paginate(:page => params[:page])
-      elsif params[:search]
-        @dogs = Dog.where('lower(name) LIKE ?', "%#{params[:search].downcase.strip}%").paginate(:page => params[:page])
-      elsif params[:status] == 'active'
-        statuses = ['adoptable', 'adoption pending', 'on hold', 'return pending', 'coming soon']
-        @dogs = Dog.where("status IN (?)", statuses).order(sort_column + ' ' + sort_direction).paginate(:per_page => 30, :page => params[:page]).includes(:photos, :primary_breed)
-      elsif params.has_key? :status
-        @dogs = Dog.where(:status => params[:status]).order(sort_column + ' ' + sort_direction).paginate(:per_page => 30, :page => params[:page]).includes(:photos, :primary_breed)
-      else
-        @dogs = Dog.where("name ilike ?", "%#{params[:q]}%").order(sort_column + ' ' + sort_direction).paginate(:per_page => 30, :page => params[:page]).includes(:photos, :primary_breed)
-      end
-    else
-      @title = "Our Dogs"
-      statuses = ['adoptable', 'adoption pending', 'coming soon']
-      @dogs = Dog.where("status IN (?)", statuses).order(sort_column + ' ' + sort_direction).paginate(:per_page => 30, :page => params[:page]).includes(:photos, :primary_breed)
-    end
-    ## Need to support the dog select drop down in the adopt app as well!
+    is_manager = signed_in? && session[:mgr_view]
+
+    @title = is_manager ? "Dog Manager" : "Our Dogs"
```

Comment:
title as in job title and not page title right?  The problem is knowing which view you're in since you can switch between a normal view and an admin / manager view